src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -6089,6 +6089,43 @@ def test_aks_disable_addon_web_app_routing(self, resource_group, resource_group_
             # self.check('ingressProfile.webAppRouting.enabled', False)
         ])
 
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westus2')
+    def test_aks_create_and_update_web_application_routing_dns_zone(self, resource_group, resource_group_location):
```

Comment:
Queued [live test pipeline](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=76371031&view=results) to validate the change.

Reply:
There's regression on command `az network dns zone` (see issue #26813), so mark the case as `@live_only()` as a temporary workaround. Will tune the case later once the regression is fixed.